Feature multi language

effofxprime opened the pull request:
Started doing a little bit.

I first tried to make more sense of the en.json by organizing it a little based on the page it would be applying to or group of items being requested.

Then I went through what you had already started and changed it to fit.  I also completed both pages at the time, but with the recent update, the governanceproposalview.vue was changed.  I will need to go back and update that one.  Since I had changes in wait, it's my main reason for wanting to get these up if you're good with the changes.

liangping commented:
Hey, great work! 
1, focus on english first. I believe someone in our community will help the other languages once we done this implement.
2. Proposal has big change since v0.46.X,  and we haven't fully compatible to the latest changes.  would like to suggest you start from other modules.

effofxprime commented:
> Hey, great work! 1, focus on english first. I believe someone in our community will help the other languages once we done this implement. 2. Proposal has big change since v0.46.X, and we haven't fully compatible to the latest changes. would like to suggest you start from other modules.

Alright, sounds good!  I honestly just wanted to edit what had already existed.  I will start to work on the other modules and leave the proposal ones alone for now.

liangping commented:
Do you want to merge this PR for now?
